augur/curate/__init__.py

```diff
@@ -45,7 +45,21 @@ def create_shared_parser():
         help="Input metadata file, as CSV or TSV.")
     shared_inputs.add_argument("--id-column",
         help="Name of the metadata column that contains the record identifier for reporting duplicate records. "
-             "Uses the first column of the metadata file if not provided.")
+             "Uses the first column of the metadata file if not provided. "
+             "Ignored if also providing a FASTA file input.")
+
+    shared_inputs.add_argument("--fasta",
+        help="Plain or gzipped FASTA file. Headers can only contain the sequence id used to match a metadata record. " +
+             "Note that an index file will be generated for the FASTA file as <filename>.fasta.fxi")
+    shared_inputs.add_argument("--seq-id-column", default="strain",
+        help="Name of metadata column that contains the sequence id to match sequences in the FASTA file.")
+    shared_inputs.add_argument("--seq-field", default="sequence",
+        help="The name to use for the sequence field when joining sequences from a FASTA file.")
```

Comment:
I just remembered that I didn't implement this check because I added default values for these options. However, looking at it now, default values don't really make sense here.